core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -99,7 +99,7 @@ public static UnderFileSystem create(String path, UnderFileSystemConfiguration u
           // when creation is done.
           Thread.currentThread().setContextClassLoader(factory.getClass().getClassLoader());
           // Use the factory to create the actual client for the Under File System
-          return new UnderFileSystemWithLogging(factory.create(path, ufsConf));
+          return new UnderFileSystemWithLogging(path, factory.create(path, ufsConf));
```

Comment:
Is it guaranteed that `path` is the mount path?

Reply:
The path is the UFS path, but not Alluxio's mounting path.

Reply:
Right, is this going to be whichever UFS path that gets accessed first, or will it specifically be the mounted UFS path (ie. `hdfs:///data` vs `hdfs:///data/2018/06/19/myfile.parquet`)?

Reply:
the former like `hdfs:///data`.

This method is called when a UFS registers, i.e. a new UFS mounts. So the path will be the url that corresponds to the mounting point.